src/components/html-injection/ko/htmlInjectionViewModelBinder.ts

```diff
@@ -0,0 +1,88 @@
+import { Bag } from "@paperbits/common";
+import { EventManager, Events } from "@paperbits/common/events";
+import { ComponentFlow, IWidgetBinding } from "@paperbits/common/editing";
+import { widgetName, widgetDisplayName, widgetEditorSelector } from "../constants";
+import { HtmlInjectionViewModel } from "./htmlInjectionViewModel";
+import { ViewModelBinder } from "@paperbits/common/widgets";
+import { StyleCompiler } from "@paperbits/common/styles";
+import { JssCompiler } from "@paperbits/styles/jssCompiler";
+import { ISettingsProvider } from "@paperbits/common/configuration";
+import { HTMLInjectionModel } from "../htmlInjectionModel";
+
+/**
+ * This class describes how the model needs to be presented (as a view model)
+ * in a specific UI framework.
+ */
+export class HtmlInjectionViewModelBinder implements ViewModelBinder<HTMLInjectionModel, HtmlInjectionViewModel>  {
+    constructor(
+        private readonly eventManager: EventManager,
+        private readonly styleCompiler: StyleCompiler,
+        private readonly settingsProvider: ISettingsProvider,
+    ) { }
+
+    public async updateViewModel(model: HTMLInjectionModel, viewModel: HtmlInjectionViewModel, bindingContext: Bag<any>): Promise<void> {
+        let htmlStyling: string = "";
+
+        if (model.inheritStyling) {
+            const environment = await this.settingsProvider.getSetting<string>("environment");
+
+            htmlStyling = '<link href="/styles/theme.css" rel="stylesheet" type="text/css">';
+
+            if (environment === "development") {
+                // Access global style sheet
+                const globalStyleSheet = await this.styleCompiler.getStyleSheet();
+
+                // Compile to CSS
+                const compiler = new JssCompiler();
+                htmlStyling += `<style>${compiler.compile(globalStyleSheet)}</style>`;
+
+                // Reflect style changes in designer
```

Comment:
Can you please remove these comments? `// TODO: looks like this is not needed after all...`